messages.js

```diff
@@ -144,13 +144,14 @@ exports.embedEventAsImage = async (event, locale) => {
 };
 
 exports.embedInventoryAsImage = async event => {
+  const l = exports.getI18n(locale);
   const good = event.good;
   const filename = `${event.EventId}-inventory.png`;
 
   return {
     embed: {
       color: good ? GREEN : RED,
-      url: KILL_URL.replace("{kill}", event.EventId),
+      url: KILL_URL.replace("{lang}", l.__("KILL_URL_LANG")).replace("{kill}", event.EventId),
```

Comment:
Thanks for your contribution!

You don't need to set a string in every locale to get the current locale. You can use l.getLocale() instead.
Please remove the entries from the locales and use the method mentioned above.